selfdrive/car/card.py

```diff
@@ -127,6 +127,17 @@ def __init__(self, CI=None, RI=None) -> None:
       safety_config.safetyModel = structs.CarParams.SafetyModel.noOutput
       self.CP.safetyConfigs = [safety_config]
 
+    secoc_key = self.params.get("SecOCKey", encoding='utf8')
```

Comment:
let's also gate this on a development build for now, like we do with alpha long support